uploader/models.py

```diff
@@ -1,12 +1,36 @@
 from django.db import models
 
+
+class SheetSize(models.TextChoices):
```

Comment:
Looks like[ the convention might be to define these classes nested within the model they implement](https://docs.djangoproject.com/en/5.0/ref/models/fields/#enumeration-types).. but I'm not sure if this matters!!

```suggestion
class Zine(models.Model):
    class SheetSize(models.TextChoices):
        ...
    
    sheet = models.CharField(max_length=4, choices=SheetSize, default=SheetSize.LETTER)
    etc.
```